client/rhel/yum-rhn-plugin/po/sk.po

```diff
@@ -9,86 +9,93 @@ msgstr ""
 "Project-Id-Version: Spacewalk\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-11-23 14:34+0100\n"
-"PO-Revision-Date: 2012-10-30 10:38+0000\n"
-"Last-Translator: adelton <jpx-tx@adelton.com>\n"
-"Language-Team: Slovak (http://www.transifex.com/projects/p/fedora/language/"
-"sk/)\n"
+"PO-Revision-Date: 2020-05-11 07:59+0000\n"
+"Last-Translator: Ferdinand Galko <galko.ferdinand@gmail.com>\n"
+"Language-Team: Slovak <https://l10n.opensuse.org/projects/uyuni/"
+"yum-rhn-plugin/sk/>\n"
 "Language: sk\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
+"X-Generator: Weblate 3.6.1\n"
 
 #: ../rhnplugin.py:48
 msgid "There was an error communicating with RHN."
-msgstr ""
+msgstr "Pri komunikácii s RHN sa vyskytla chyba."
 
 #: ../rhnplugin.py:63
 msgid "Red Hat Satellite or RHN Classic support will be disabled."
-msgstr ""
+msgstr "Podpora Red Hat Satellite alebo RHN Classic bude zakázaná."
 
 #. If non-root notify user Spacewalk repo not accessible
 #: ../rhnplugin.py:70
 msgid ""
 "*Note* Spacewalk repositories are not listed below. You must run this "
 "command as root to access Spacewalk repositories."
 msgstr ""
+"*Poznámka* Repozitáre Spacewalk nie sú nižšie uvedené. Tento príkaz musíte "
+"spustiť ako správca systému, aby ste získali prístup k repozitárom Spacewalk."
```

Comment:
Isn't it `správca systému` translated as `system administrator`? Of course I am not slovak speaker, but I wonder if it's int more clear to talk about the `root user` to make it clear.